cmd/producer/main.go

```diff
@@ -23,17 +23,17 @@ func main() {
 	// Get Config
 	config, err := config.Get()
```

Comment:
The variable name config clashes with the package name so better to name it cfg or something like that e.g.
```suggestion
	cfg, err := config.Get()
```